### PR TITLE
fix(destination): updated bing ads syntax as per the updated Microsoft's syntax

### PR DESCRIPTION
**Pull request by sandeepdsvs**
## Description of the change

> Microsoft updated the syntax for the javascript UET tracking library. Ref [doc](https://help.ads.microsoft.com/#apex/ads/en/56916/2-500). So made changes in bind ads destination reflecting the same.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/740)

### CODE REVIEW

File: integrations/BingAds/browser.js [hunk 1]
```diff
@@ -58,23 +58,23 @@ class BingAds {
     const { type, properties, event } = rudderElement.message;
     const { category, currency, value, revenue, total } = properties;
     const payload = {
-      ea: type,
-      el: event,
+      event: type,
```
Comment (utsabc) [inline integrations/BingAds/browser.js:61]: Can we add some destination ref page link in here

Comment (sandeepdsvs) [reply integrations/BingAds/browser.js:63]: updated

**Review by ItsSudip — APPROVED** [review]
Can we add support for [event_value](https://help.ads.microsoft.com/#apex/ads/en/56709/2:~:text=trailer%27%20or%20%27behindthescenes.%E2%80%99-,Event%20value,-%3A%20A%20numerical%20value) as well?

**Review by utsabc — APPROVED** [review]

**Comment by sandeepdsvs** [comment]
Will raise a new PR for event_value support cc @ItsSudip

**Pull request merged**